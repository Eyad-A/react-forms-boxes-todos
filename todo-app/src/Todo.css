button {
    position: relative;
	display: inline-block;
	padding: 5px;
	color: #FFF;
	margin: 10px 20px;
	background: #c41717;
	font-size: 12px;
	text-align: center;
	font-style: normal;	
}

