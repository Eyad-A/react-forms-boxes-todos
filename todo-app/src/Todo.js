import React from "react";

function Todo({ task = "Walk the dog", id = "1", remove}) {
    const handleDelete = () => remove(id);
   
    return (
        <div>
            <div>
                <li>{task}</li>
            </div>
            <button onClick={handleDelete}>X</button>
        </div>
    );
}

export default Todo;