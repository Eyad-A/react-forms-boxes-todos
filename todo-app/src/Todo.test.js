import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Todo from "./Todo";

it("renders without crashing", function() {
    render(<Todo />);
});

it("should match the snapshot", function() {
    const {asFragment} = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
});

