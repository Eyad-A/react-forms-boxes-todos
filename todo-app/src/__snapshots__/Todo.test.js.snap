// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <li>
        Walk the dog
      </li>
    </div>
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
